run_on_cluster

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Convenient script to run gemini on cluster
+
+# N.B:
+# make sure kubectl has the same version as server
+# for MacOs remove client and install like this:
+# brew install https://raw.githubusercontent.com/Homebrew/homebrew-core/a9f7c11c63db20f3746e034849559ef266808cd1/Formula/kubernetes-cli.rb
+
+# create pod if not exists
+if ! kubectl get pods | grep gemini; then
+    kubectl create -f ./k8s/gemini-pod.yaml
+fi
+
+while ! kubectl get pods | grep "gemini.*Running"; do
+    sleep 1
+done
+
+# variables
+DB_HOST=scylladb
+
+case $1 in
+"hash")
+    kubectl exec gemini -- /bin/bash -c "MASTER=spark://p-spark-master:7077 /gemini/hash -h $DB_HOST ${@:2}"
+    ;;
+"query")
+    kubectl exec gemini -- /gemini/query -h $DB_HOST "${@:2}"
+    ;;
+"report")
+    kubectl exec gemini -- /gemini/report -h $DB_HOST "${@:2}"
+    ;;
+*)
+    # just get into container
+    kubectl exec -it gemini /bin/bash
```

Comment:
`run_on_cluster reportt` will open you a bash into the container?

I think the following could be useful:
- `run_on_cluster bash` &rarr; open a shell
- `run_on_cluster help | whatever-typo | nothing` &rarr; show the available commands.

How would you see that?

Reply:
By the way, I like the idea of having `wrapper-command <old-comand> <same old parameters>`, so you can run the same thing keeping the api of the `<old-command>`.
LQCTM (Looks Quite :sunglasses: To Me )

Reply:
it does exactly that already. Like `./run_on_cluster hash -v -f bare hdfs://path`
But it will set up DB & spark master for you and run it inside a container instead of local.

Reply:
Yes, I like it a lot

Reply:
ahahaha. I thought you suggested this improvement, not just pointed on it. :-D Sorry.